src/Chapter1.hs

```diff
@@ -641,8 +670,11 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
-
+firstDigit :: Int -> Int
+firstDigit n = 
+    if (div (abs n) 10) == 0 
+    then (abs n) 
+    else firstDigit (div (abs n) 10) 
```

Comment:
Perfect! That is a very elegant implementation 💃🏼 

Side note, you can see that you need to use `abs n` in several different places. Usually it the sign that you can move it into the variable. In this case, you can put it into the `where` clause and reuse everywhere 🙂

Reply:
> Perfect! That is a very elegant implementation 💃🏼



Thank you. :)

Made the suggested changes.